app/src/main/java/org/onionshare/android/server/SendPage.kt

```diff
@@ -0,0 +1,32 @@
+package org.onionshare.android.server
+
+data class SendPage(
+    private val fileName: String,
+    private val fileSize: String,
+    private val fileSizeHuman: String,
+    private val title: String = "OnionShare",
+) {
+    private val files: ArrayList<SendFile> = ArrayList()
+    val model: Map<String, Any>
+        get() {
+            return mapOf(
+                "filename" to fileName,
+                "filesize" to fileSize,
+                "title" to title,
+                "filesize_human" to fileSizeHuman,
+                "is_zipped" to "1",
+                "files" to files,
+//                "download_individual_files" to "",
+//                "dirs" to dirs,
+            )
```

Comment:
The template engine (which is *mostly* compatible with what the python app uses) requires things to be passed in as a map. I listed all values that are used and commented out the ones, we are not supporting for now.

Note that I duplicated the entries as fields to get type-safety.